enterprise/users/src/main/java/io/crate/metadata/UsersMetaData.java

```diff
@@ -189,4 +190,9 @@ public int hashCode() {
     public String getWriteableName() {
         return TYPE;
     }
+
+    @Override
+    public Version getMinimalSupportedVersion() {
+        return Version.V_6_0_0;
```

Comment:
We probably have to think about how we make use of this in the future. For now I chose `6.0.0` everywhere. We shipped CrateDB 3.0 with ES 6.1 and it's not supported to do rolling upgrades from earlier versions to it. So this should be safe.